### DISCUSSION TITLE
What happened to the "Aggregate Files" function?

### DISCUSSION THREAD

**Discussion by Maxmillium** [aclap-dev/video-downloadhelper#1282] [Help and Q&A]
### Your question

VDH Version 9.1.0.41 (beta), target mozilla, locale en-US.
Companion App installed (v2.0.19)
running on 
Linux - OpenSuse Leap 15.4
Firefox 115.6.0esr 64-bit

I used to be able to aggregate the audio and video versions of the same video when they were the only files that would download.
Where is the "Aggregate Files" function in the current version of Video-DownloadHelper?

### Addon & CoApp version and details

version: 9.1.0.41
channel: beta
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/admin/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/admin",
  "converterBinary": "/home/admin/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I know you know that.  You're an old-timer from way back.  I recognize your user ID.  But I like to say that so anybody stumbling on this discussion has no confusion over that fact.

Paul purposely did not port that functionality forward to VDH 9.  Sad but true.  But it's so simple to merge a video track with an audio track.  I show how in #1243.  You may have to read a bit but there is an explanation of merging tracks in there.

**Reply by Maxmillium** [reply]
Willy:  Is there a place on this site for simple conversation without  cluttering the vital functions of the site?
Maximillium

**Reply by GrampaWildWilly** [reply]
You mean like a chill-out area?  Just for chitchat without any specific technical purpose?  I don't know.  I don't think there's one here in the VDH forum.  There might be one under the GitHub umbrella somewhere.  I've never looked.  But you could.

**Reply by GrampaWildWilly** [reply]
https://github.com/orgs/community/discussions?discussions_q=

Looks like there isn't much frivolity there.

**Comment by Maxmillium** [comment]
Looks like we're already in the category "Discussions".
Anyway, good to talk with you, too....

**Comment by Maxmillium** [comment]
Yup.  I guess just use the aclap-dev page when there's something to say.
I didn't want to use this for other-than-on-topic anything, but no way to 
privately exchange email addresses.
Not that much to say anyway ... except right now - I really want to see that 
"aggregate" function back in the program!

-- 
  OLFarCnArt
  ***@***.***

**Reply by Mattmjs** [reply]
You can do that in VLC , I saw a video on youtube and it works.
